src/compas_eve/core.py

```diff
@@ -64,7 +79,21 @@ def parse(cls, value):
 
 
 class Topic(object):
-    """Describes a topic"""
+    """A topic is like a mailbox where messages can be sent and received.
+
+    Topics are described by a name, a type of message they accept, and a
+    set of options.
+
+    Attributes
+    ----------
+    name : str
+        Name of the topic.
+    message_type : type
+        Class defining the message structure. Use :class:`Message` for
+        a generic, non-typed checked message implementation.
+    options : dict
```

Comment:
maybe an example of some options in `examples`?

Reply:
right now, there are no options implemented, so, I just added a `TODO` note https://github.com/compas-dev/compas_eve/pull/4/commits/42a33e53d36a57ed517feb640d4f6f15dfa25830